### PR TITLE
Added a computer science question

### PR DESCRIPTION
**Pull request by tanbirali**
## Checklist

- [x] I have read and followed the [contribution guidelines](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CONTRIBUTING.md).
- [x] I have read through the [Code of Conduct](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CODE_OF_CONDUCT.md) and agree to abide by the rules.
- [x] This PR is for one of the available issues and is not a PR for an issue already assigned to someone else.
- [x] My PR title has a short descriptive name so the maintainers can get an idea of what the PR is about.
- [ ] I have provided a summary of my changes.

closes #XXXXX

### CODE REVIEW

File: src/data/general-cs-quiz.ts [hunk 1]
```diff
@@ -19,6 +19,16 @@ const generalCSQuiz = [
     Explanation:
       "In the call-by-value approach, a replica of the argument's value is generated, and any modifications made to the formal parameter within the function do not affect the original argument outside of the function, as the function operates on its own isolated copy of the data.",
     Link: "https://www.freecodecamp.org/news/understanding-by-reference-vs-by-value-d49139beb1c4/"
+  },
+   {
```
Comment (jdwilkin4) [inline src/data/general-cs-quiz.ts:23]: fix formatting here

```suggestion
  {
```

File: src/data/general-cs-quiz.ts [hunk 2]
```diff
@@ -19,6 +19,16 @@ const generalCSQuiz = [
...
+    Question: "Which of hte following algorithms is used to find the shortest path between two points in a connected weighted graph?",
```
Comment (jdwilkin4) [inline src/data/general-cs-quiz.ts:24]: spelling error

```suggestion
    Question: "Which of the following algorithms is used to find the shortest path between two points in a connected weighted graph?",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by tanbirali** [comment]
@jdwilkin4 I have committed the changes you asked me to make please let me know if it done right.

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**